root/components/FieldErrors.js

```diff
@@ -9,25 +9,34 @@
 
 import React from 'react';
 
+import expand2react from '../static/scripts/common/i18n/expand2react';
 import subfieldErrors, {type FieldShape} from '../utility/subfieldErrors';
 
-const buildErrorListItem = (error, index) => (
-  <li key={index}>{error}</li>
-);
+const buildErrorListItem = (error, hasHtmlErrors, index) => {
```

Comment:
```suggestion
// FIXME: Convey expandable object instead of HTML string for safety, see MBS-10632.
const buildErrorListItem = (error, hasHtmlErrors, index) => {
```